Airline_Web_API/Airline_Web_API/Controllers/TicketTypesController.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Airline_Web_API.Helpers;
+using Airline_Web_API.Services;
+using Airline_Web_API.ViewModels;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion.Internal;
+
+namespace Airline_Web_API.Controllers
+{
+    [Route("api/ticket-types")]
+    [ApiController]
+    public class TicketTypesController : ControllerBase
+    {
+        private readonly TicketTypeService _ticketTypeService;
+        public TicketTypesController(TicketTypeService ticketTypeService)
+        {
+            _ticketTypeService = ticketTypeService;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<TicketTypeViewModel>>> GetTicketTypes()
+        {
+            var ticketTypes = await _ticketTypeService.GetTicketTypesAsync();
+
+            if (ticketTypes == null)
+            {
+                return NoContent();
+            }
```

Comment:
More common alternative for endpoints that return collection will be just returning empty collection.
That way you don't need ActionResult.
BTW, is it possible to get null in ticketTypes?

Reply:
Oh, this chick is unnecessary here. I've corrected it